server/app.py

```diff
@@ -68,10 +78,12 @@ class Cache:
   stops: typing.List[Stop]
   updated_at: str
 
+pst = pytz.timezone('America/Los_Angeles') 
 PREDICTIONS_URL = 'https://api.511.org/transit/StopMonitoring'
 cache = Cache([], '')
 def update_cache():
     new_stops = []
+    cache.updated_at = datetime.datetime.now(datetime.timezone.utc).astimezone(pst).time()
```

Comment:
can we keep everything in utc? i dont think we need pytz anymore then

Reply:
resolved